daemon/process/settings/dobby.dev_vm.json

```diff
@@ -1,12 +1,12 @@
 {
     "dbus": {
-      "serviceName": "com.sky.dobby",
-      "objectPath": "/com/sky/dobby"
+      "serviceName": "org.rdk.dobby",
+      "objectPath": "/org/rdk/dobby"
```

Comment:
We should reflect this change in the OCIContainers Thunder plugin as well, where we're still using `com.sky.dobby.test`.